Add AES support to netlogon

commit 2312194e96d9501549bff6c285ddfae82515e963
Author: etxrab <etxrab@f5534014-38df-0310-8fa8-9805f1628bb7>
Date:   Sun Mar 4 16:07:24 2012 +0000

    "From Jan Šafránek: Add support for AES in Microsoft Network Logon Negotiation options https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=6895"
    
    git-svn-id: http://anonsvn.wireshark.org/wireshark/trunk@41337 f5534014-38df-0310-8fa8-9805f1628bb7

diff --git a/epan/dissectors/packet-dcerpc-netlogon.c b/epan/dissectors/packet-dcerpc-netlogon.c
index 859334a..04d494f 100644
--- a/epan/dissectors/packet-dcerpc-netlogon.c
+++ b/epan/dissectors/packet-dcerpc-netlogon.c
@@ -6671,12 +6671,12 @@ static int netlogon_dissect_neg_options(tvbuff_t *tvb,proto_tree *tree,guint32 f
           hf_netlogon_neg_flags_2000000,
           tvb, offset, 4, flags);
           proto_tree_add_boolean (negotiate_flags_tree,
-          hf_netlogon_neg_flags_1000000,
-          tvb, offset, 4, flags);
-          proto_tree_add_boolean (negotiate_flags_tree,
           hf_netlogon_neg_flags_800000,
           tvb, offset, 4, flags);*/
         proto_tree_add_boolean (negotiate_flags_tree,
+                                hf_netlogon_neg_flags_1000000,
+                                tvb, offset, 4, flags);
+        proto_tree_add_boolean (negotiate_flags_tree,
                                 hf_netlogon_neg_flags_400000,
                                 tvb, offset, 4, flags);
         proto_tree_add_boolean (negotiate_flags_tree,
@@ -8821,7 +8821,7 @@ proto_register_dcerpc_netlogon(void)
           { "Not used 2000000", "ntlmssp.neg_flags.na200000", FT_BOOLEAN, 32, TFS(&tfs_set_notset), NETLOGON_FLAG_2000000, "Not used", HFILL }},
 
         { &hf_netlogon_neg_flags_1000000,
-          { "Not used 1000000", "ntlmssp.neg_flags.na100000", FT_BOOLEAN, 32, TFS(&tfs_set_notset), NETLOGON_FLAG_1000000, "Not used", HFILL }},
+          { "AES supported", "ntlmssp.neg_flags.na100000", FT_BOOLEAN, 32, TFS(&tfs_set_notset), NETLOGON_FLAG_1000000, "AES", HFILL }},
 
         { &hf_netlogon_neg_flags_800000,
           { "Not used 800000", "ntlmssp.neg_flags.na8000000", FT_BOOLEAN, 32, TFS(&tfs_set_notset), NETLOGON_FLAG_800000, "Not used", HFILL }},
